2-mandatory/4-stocks.js

```diff
@@ -49,8 +60,17 @@ function getAveragePrices(closingPricesForAllStocks) {
 */
 function getPriceChanges(closingPricesForAllStocks) {
     // TODO
+    let priceChangeStock = [];
+    for (let stockPriceBrand of closingPricesForAllStocks) {
+      let priceChangeBrand = stockPriceBrand[4] - stockPriceBrand[0];
+      let FormattedPriceChangeBrand = parseFloat(priceChangeBrand.toFixed(2));
```

Comment:
try to use `const` where possible - also in general functions and variables should be `lowerCamelCase`, `UpperCaselCase` (also called `PascalCase`) is reserved for `class`es and `type`s